package.json

```diff
@@ -1,19 +1,22 @@
-{
-  "name": "mmm-discordwatch",
-  "version": "0.1.0",
-  "description": "MagicMirror2 module for watching discord channels.",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "keywords": [
-    "MMM",
-    "magicmirror",
-    "discord"
-  ],
-  "author": "Smiche",
-  "license": "CC Zero",
-  "dependencies": {
-    "discord.js": "^12.5.1"
-  }
-}
+{
+  "name": "mmm-discordwatch",
+  "version": "0.1.0",

```

Comment:
can you bump to 0.2.0 so the old package is still available to people using discordjs 12

Reply:
ok.
Changed the version to 0.2.0.